src/applications/check-in/api/versions/v2.js

```diff
@@ -74,10 +74,11 @@ const v2 = {
     };
   },
 
-  getCheckInData: async token => {
+  getCheckInData: async (token, reload = false) => {
     const url = '/check_in/v2/patient_check_ins/';
+    const typeSlug = reload ? '?reload=true' : '?reload=false';
```

Comment:
NITPICK: I don't think `typeSlug` is a good variable name, I would consider something with reload like `reloadSlug`